lib/templates/middleAssignment/local_storage.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:hive_flutter/hive_flutter.dart';
+import 'models/session_model.dart';
+import 'package:intl/intl.dart';
+
+
+
+class LocalStorageService {
+  LocalStorageService() {
+    _initHive();
+  }
+
+  Future<void> _initHive() async {
+    await Hive.initFlutter();
+    Hive.registerAdapter(SessionAdapter());
+    await Hive.openBox("sessions");
+    // await Hive.box('sessions').clear();
```

Comment:
<img width="293" alt="Screenshot 2024-06-25 at 08 25 18" src="https://github.com/timur-harin/Sum24FlutterCourse/assets/64702874/95d3267d-f9b6-49fc-842b-def6ec840d3d">
always not ended circle